packages/dev/core/src/Materials/Node/nodeMaterialConnectionPointCustomObject.ts

```diff
@@ -19,7 +19,8 @@ export class NodeMaterialConnectionPointCustomObject<T extends NodeMaterialBlock
         name: string,
         ownerBlock: NodeMaterialBlock,
         direction: NodeMaterialConnectionPointDirection,
-        private _blockType: new (...args: any[]) => T,
+        // @internal
+        public _blockType: new (...args: any[]) => T,

```

Comment:
Is there no parent class for all block types?

Reply:
The thing here is that this is the blockType constructor, and I didn't want to change typing to avoid affecting anything else 😬